testing/test_doctest.py

```diff
@@ -146,9 +146,9 @@ def test_encoding_ascii(self, testdir):
     def test_encoding_latin1(self, testdir):
         """Test support for --doctest-encoding option.
         """
-        testdir._makefile(".txt", ["""
-            >>> 'üäö'
-            'üäö'
+        testdir._makefile(".txt", [u"""
+            >>> len(u'üäö')
+            3
```

Comment:
In Python 2, they will be formatted as escape sequences: `u'\x84\x94\x81'`. Probably I can just `repr(value)` to the files.